src/Api/Api.ts

```diff
@@ -1,75 +1,74 @@
 import { Method, HeadersMap, Response } from './types';
 import createRequest from './createRequest';
 
+const withAccessToken = (headers?: HeadersMap, accessToken?: string): HeadersMap => ({
+    ...headers,
+    ...(accessToken ? { 'x-access-token': accessToken } : {}),
+});
+
+interface Params {
+    accessToken?: string;
```

Comment:
Should it not be required?

```suggestion
    accessToken: string;
```

Reply:
78fb90580c962f9d001fc254714f6408ef6ba61a